ReactSkia/core_modules/RSkAlertManager.cpp

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include "RSkAlertManager.h"
+
+#include "sdk/sdkStylesConfig.h"
+
+namespace facebook {
+namespace react {
+
+std::mutex lockMsgDrawing;
```

Comment:
should this be global or part of RSkAlertManager ?

Reply:
Currently, we are keeping this as part of RSkALertManager only.